api/.env.dev

```diff
@@ -6,12 +7,14 @@ POSTGRES_DB="hub"
 POSTGRES_USER="postgres"
 POSTGRES_PASSWORD="postgres"
 
-GH_CLIENT_ID="client-id"
-GH_CLIENT_SECRET="client-secret"
+GITHUB_ID=""
```

Comment:
it might create confusion with github id (user id)

Reply:
So you mean we should keep as GH_CLIENT_ID ??

Reply:
yes , WHYT?

Reply:
so for gitlab it will be GL_CLIENT_ID and bitbucket BB_CLIENT_ID ??

Reply:
yes